src/webgpu/api/operation/copyBetweenLinearDataAndTexture.spec.ts

```diff
@@ -364,6 +364,7 @@ class CopyBetweenLinearDataAndTextureTest extends GPUTest {
       expectedDataLayout,
```

Comment:
Note: expectedDataLayout has an `offset` in it. It was previously obeyed on L364 but ignored on L363. It is now obeyed in both places due to the change to `updateLinearTextureDataSubBox`.